tasks/receptionist/src/receptionist/states/find_and_look_at.py

```diff
@@ -0,0 +1,269 @@
+"""
+State machine for the nicole task. It finds a person by given name and then looks at them.
```

Comment:
nicole task? :D

Reply:
hahah cringe

Reply:
done (dont work on pycharm and let it edit freely...)